### PR TITLE
updated

### PR DESCRIPTION
**Pull request by ninakoo289**
Updated

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ingcognito) [inline .vscode/settings.json:1]: this is good, but only works on your computer because you use Visual Studio Code

File: WorkEx.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<body>
+
+<div class="jumbotron text-center">
+  <h1>My Work Experience</h1>
+  <p>Stay Patient And Trust Your Journey</p> 
+</div>
+  
+<div class="container">
+  <div class="row">
+    <div class="col-sm-4">
```
Comment (ingcognito) [inline WorkEx.html:20]: review https://www.w3schools.com/bootstrap/bootstrap_grid_system.asp

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Contact</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (ingcognito) [inline contact.html:8]: https://www.w3schools.com/js/js_ajax_intro.asp

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (ingcognito) [inline contact.html:9]: https://www.webopedia.com/TERM/C/CDN.html#:~:text=CDN%20is%20short%20for%20content,and%20the%20content%20delivery%20server.

File: index.html [hunk 5]
```diff
@@ -63,12 +63,13 @@
...
       </button>
       <a class="navbar-brand" href="#">ME</a>
+     
     </div>
     <div class="collapse navbar-collapse" id="myNavbar">
       <ul class="nav navbar-nav navbar-right">
-        <li><a href="#">EDUCATION</a></li>
-        <li><a href="#">WORK EXPERIENCE</a></li>
-        <li><a href="#">CONTACT</a></li>
+        <li><a href="http://127.0.0.1:5501/education.html#">EDUCATION</a></li>
```
Comment (ingcognito) [inline index.html:70]: you hardcoded the URL here. This should direct to local file

**Comment by ingcognito** [comment]
please resolve the comments and you're good to merge

File: .vscode/settings.json (hunk 1 above)
Comment (ninakoo289) [reply .vscode/settings.json:1]: That's right. 

I known a lot from you.

File: WorkEx.html (hunk 2 above)
Comment (ninakoo289) [reply WorkEx.html:20]: Thanks!

File: contact.html (hunk 3 above)
Comment (ninakoo289) [reply contact.html:8]: 👍

File: index.html (hunk 5 above)
Comment (ninakoo289) [reply index.html:70]: 💯

**Pull request merged**